packages/service-clients/end-to-end-tests/azure-client/package.json

```diff
@@ -26,6 +26,7 @@
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
 		"start:tinylicious:test": "PORT=7071 tinylicious > tinylicious.log 2>&1",
 		"test": "npm run test:realsvc",
+		"test:azureclient:tinylicious:report": "npm run test:realsvc:tinylicious",
```

Comment:
I see this is doing the same thing as `test:realsvc`, which is already run by `test` - maybe don't need the duplication?

Reply:
The idea was to give it a unique name so that when we run it in the build pipeline it is obvious what we are running, see [here](https://dev.azure.com/fluidframework/public/_build/results?buildId=263659&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=ddd9efee-bc1d-59c0-82a9-543538b126ce)

Reply:
Your call (or maybe someone who thinks more about test scripts will have some opinions) - I don't have a strong opinion here, so just presenting some alternative ideas.